test.js

```diff
@@ -946,7 +916,7 @@ test('createAsyncIterator creates Iterator for Array', () => {
 })
 
 test('createAsyncIterator creates Iterator for Iterator', () => {
-  var myIterator = getIterator(['Alpha', 'Bravo', 'Charlie'])
+  var myIterator = getIterator((['Alpha', 'Bravo', 'Charlie']: Iterable<string>))
```

Comment:
This is definitely a bug in Flow. However, I think this can be fixed relatively easily with some core type definitions, I'll look into it.